server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/WhiteListManager.java

```diff
@@ -1,793 +0,0 @@
-/****************************************************************
```

Comment:
After a second though, to remove this class we need to be able to make the droplist work with the JPA app with a SQL backend as otherwize James JPA users would not be able to have an alternative.

Maybe we can implement a JPA backend prior to the removal? WDYT?

Reply:
Yes I agree. Before deleting, I'll try to implement a JPA backend.